### PR TITLE
JOgle_hw1

### PR DESCRIPTION
**Pull request by jackocyber**
I added my name, a few other facts about myself, and my taste in movies to the README. At first I found it a little challenging to get the R studio syntax down. But I found GitHub Guides to be very helpful in understanding the syntax. Additionally, the video on the course website was also very helpful in getting down the assignment workflow. There are many steps in the process, so it was good to have a broad explanation first then a more detailed explanation. After watching the video I was able to better understand the written text on the assignment page. Overall, I enjoyed this first assignment and it was not overly challenging. Going forward, I will start the assignments early as I did this one. This is because I know they will get harder and more complex requiring more time.

### CODE REVIEW

**Comment by wmccullen** [comment]
Very good job, everything works! I like how you used multiple images and took the list a step further!

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+
+
+> # "Anyone Can Cook""
+> ## - Chef Gusteau 
+
+![Chef Gusteau and Remy](https://user-images.githubusercontent.com/65682542/85363397-749ac800-b4e6-11ea-8ffb-5eab32255b88.jpg)
```
Comment (bensoltoff) [inline README.md:6]: Nice choice! And how appropriate for this course

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+
+
+# My name is Jack Ogle and I have been rewatching all the pixar movies during quarentine!
+
+## _*This is me*_
+!["I think I see a bald eagle!!!!"](https://user-images.githubusercontent.com/65682542/85432720-f9b6c900-b548-11ea-9c72-814d2e96f73c.jpg)
```
Comment (bensoltoff) [inline README.md:12]: I recommend leaving a blank line between different Markdown elements, such as between headers and paragraph text. It avoids rendering issues in more complex documents

File: README.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+1. Ratatouille
+    * *A fantastically unconventional idea combinded with perfectly timed jokes!*
+1. Incredibles
+    * *The whole family dynamic is raw and real, something you rarly get from the typical super hero fare.*
+1. Monsters Inc.
+    * *Billy Crystal and John Goodman have a chemistry that rivals Amedeo Avogadro. Amedeo Avogadro as in Avogadro's number.*
+1. Toy Story
+    * *Nostalgia out the wazoo, need I say more?*
+1. Up
+    * *The first fifteen minutes get me every- dang-time.*
```
Comment (bensoltoff) [inline README.md:27]: Great use of numbered lists

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Reproducibility**         |    x   x    |              |            |

**Comment by sabreena-croteau** [comment]
I liked the visuals and I thought the headers and the list were really well done. You didn't really talk about your process and I would have liked to know, for my own curiosity.

**Pull request closed without merging**